drag-and-drop.md

```diff
@@ -0,0 +1,126 @@
+# Drag-adn-Drop
+
+The proposal to provide native support Drag-and Drop opeerations:
+* drag and drop between widgets 
+* receive drag and drop URL(s) to application from a desktop or another application
+* start drag operation from Fyne application with URL(s) to another desltop application (pending external changes)
+
+## Background
+
+There are few issues opened and related to missing drag and drop functionality:
+* Add drag and drop support https://github.com/fyne-io/fyne/issues/142
+* Drag and drop to application icon on macOS https://github.com/fyne-io/fyne/issues/497
+
+Existing interface provide a minimum capabilities to programmatically start a drug operation and get notified when it complets.
+This functionality doesn't provide an interface to communicate between widgets within the application nor send/receive drag and drop with other applications.
+
```

Comment:
I would probably add notes here about how macOS does not provide file arguments on the cmdline like windows and linux does? I know you link it above, but got to summarise as well.